app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -315,9 +322,14 @@ private void loadActivityHeaderPicture() {
         imagePath = activity.getDocumentPath() + "/" + ACTIVITY_IMAGE_NAME;
         Image image = new Image(null, activityImage);
         image.setDocumentPath(imagePath);
-        ImageHandler.loadImage(image, progressBar);
+        ImageHandler.loadImage(image, this);
     }
 
+    public ProgressBar getProgressBar() {
```

Comment:
Ok ok, that's what you said, since you changed the signature of the methods in the `ImageHandler` class which now take as argument the whole activity, you had to create a getter for the progress bar in `ActivityDescriptionActivity`, `ActivityDescriptionActivityUnregister`, `EditProfileActivity`, etc., I see 👍.

Reply:
Exact ! It is necessary to have it for the permission checker, so I can directly access all the elements needed with getters. But, there is still a bug with  the progress bar that doesn't seem to always appear...